projects/plutus/test/test_ltc2484.c

```diff
@@ -0,0 +1,93 @@
+#include <stddef.h>
```

Comment:
All these tests are for the parser. I guess I could write a couple tests for hardware, but they'd assume that we're running current one direction (or maybe the test assumes no current is running? idk).

Reply:
Nah, I think testing the parser is enough. I would probably add a test project so we can easily verify the current sensor once we get isoSPI fixed. Characterize ripple, test 0A, -5A, +5A?

Reply:
Should I just overwrite `main` with that?

I can't really create a new project since it would need the drivers, which are under the `plutus` project (unless we're okay with duplicating the drivers?).